src/ck-core/debug-charm.C

```diff
@@ -866,6 +874,12 @@ void CpdContinueFromBreakPoint ()
           
         }
     }
+    /*        else
```

Comment:
This could be `#if 0` instead of a block comment

Reply:
done in fixup